.circleci/config.yml

```diff
@@ -1,69 +0,0 @@
----
-common-steps:
-
-  - &installdeps
-    run:
-      name: Install Debian packaging dependencies
-      command: |
-        apt-get update && apt-get install -y make sudo
-        make install-deps
-
-  - &installtestdeps
-    run:
-      name: Install test dependencies
-      command: |
-        apt-get install reprotest faketime -y --no-install-recommends
-        .venv/bin/pip install -r test-requirements.txt
-
-
-version: 2.1
-
-jobs:
-  lint-and-test:
-    docker:
-      - image: debian:bullseye
-    steps:
-      - checkout
-      - *installdeps
-      - *installtestdeps
-      - run:
-          name: install test requirements, run linters, and run tests
-          command: |
-            source .venv/bin/activate
-            sed -i -re "292s/^(\s+).*\$/\1return _.prepend_to_build_command_raw('')/" /usr/lib/python3/dist-packages/reprotest/build.py
-            make test
-
-  reprotest-wheels:
-    docker:
-      - image: debian:bullseye
-    # Our "ci" user will have problems if the repository is in /root
-    working_directory: "/srv"
-    steps:
-      - checkout
-      - *installdeps
-      - *installtestdeps
-      - run:
-          name: install test requirements and run tests
-          command: |
-            adduser --system ci --ingroup root
-            sed -i -re "292s/^(\s+).*\$/\1return _.prepend_to_build_command_raw('')/" /usr/lib/python3/dist-packages/reprotest/build.py
```

Comment:
This in-place patching goes back to 18770bd44ec54da86f436515da4452c05111605e and, if it was still relevant at all, was not necessary for the wheel reproducibility checks, which do not use the `reprotest` package that is being patched here. The actual Debian reproducibility check has been migrated into `securedrop-client` and no longer uses `reprotest`.